### PR TITLE
Remove additional effect

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1161
[Problem] - When button is clicked longer than usual,
there is additional pulse animation
[Solution] - Remove additional animation

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
Is this PR for issue https://github.com/Samsung/TAU/issues/1161 or just part of it?
As mentioned in doc
image
and as you can see in sample app
image

issue is about ripple effect not removed after long press.
After this PR I can still see this issue

File: src/css/profile/mobile/common/button.less [hunk 1]
```diff
@@ -210,8 +210,7 @@ a.ui-btn {
 			&::before {
 				opacity: 1;
 				animation: btn_press_animation_flat_opacity_in linear 100ms,
-					btn_press_animation_flat_opacity_out linear 400ms 100ms,
-					btn_press_animation_flat_scale cubic-bezier(0.33, 0, 0.2, 1) 350ms;
+					btn_press_animation_flat_opacity_out linear 400ms 100ms;
```
Comment (lslachciak) [inline src/css/profile/mobile/common/button.less]: this is not needed now

Comment (DohyungLim) [inline src/css/profile/mobile/common/button.less]: I agree with @lslachciak .
It is better not to modify it unless it is functionally necessary.

**Review by DohyungLim — COMMENTED** [review]
1 comment.

**Review by lslachciak — COMMENTED** [review]
works ok for me, but please double check if this has no side effects for other type of buttons

**Comment by korneliakobiela** [comment]
Checked, I didn't observe any side effects.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**